app/views/content/service_privacy_notice.md

```diff
@@ -81,6 +81,7 @@ Processing applications includes:
 - getting in touch with you to ask if you would like to participate in user research
 - getting in touch with you and providers about your applications
 - analysing application and service usage data
+- researching and analysing in conjunction with education providers' information
```

Comment:
Could you say 'using information from education providers and our own to research and analyse how candidates apply for initial teacher training'?

Reply:
I like this - I've passed this on to Katie this morning, so will feed back tomorrow when she's working.

Reply:
@MaeveRoseveare here's Katie's reply:

I wonder if we could be broader and say 'using information from education providers and information from the Apply service for analysis and research about initial teacher training'

Reply:
Sounds good, normally I'd say drop the 'service', however I think in this case it makes it clearer on what we're talking about. So happy with that!